community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/NodeCountsProcessor.java

```diff
@@ -49,6 +51,18 @@ public NodeCountsProcessor( NodeStore nodeStore, NodeLabelsCache cache, int high
         this.labelCounts = new long[highLabelId + 1];
     }
 
+    public NodeCountsProcessor( NodeStore nodeStore, NodeLabelsCache cache, int highLabelId,
```

Comment:
This one can be package-private and the other constructor is not used by anyone and can be removed.